backend/python/app/tests/helpers/story_translation_helpers.py

```diff
@@ -0,0 +1,141 @@
+    )
+    assert story_translation_response["story_id"] == story_translation_model.story_id
+    assert story_translation_response["title"] == story_model.title
+    assert story_translation_response["description"] == story_model.description
+    assert story_translation_response["youtube_link"] == story_model.youtube_link
+    assert story_translation_response["level"] == story_model.level
+    assert story_translation_response["num_translated_lines"] == num_translated_lines
+    assert story_translation_response["num_approved_lines"] == num_approved_lines
+    assert story_translation_response["num_content_lines"] == len(translation_contents)
+
+
+def create_story(db):
+    return db_session_add_commit_obj(
+        db,
+        Story(
+            title="East of Eden",
+            description="Follow the intertwined destinies of two families whose generations reenact the poisonous rivalry of Cain and Abel.",
+            youtube_link="https://www.youtube.com/watch?v=redECmF7wh8",
+            level=4,
+        ),
+    )
+
+
+def create_translator(db):
+    return db_session_add_commit_obj(
+        db,
+        UserAll(
+            first_name="Carl",
+            last_name="Sagan",
+            email="carl.sagan@uwblueprint.org",
+            auth_id="secret",
+            role="User",
+            approved_languages_translation={"ENGLISH_US": 4, "ENGLISH_UK": 4},
+            approved_languages_review={"ENGLISH_US": 4, "ENGLISH_UK": 4},
+        ),
+    )
+
+
+def create_reviewer(db):
+    return db_session_add_commit_obj(
+        db,
+        UserAll(
+            first_name="Dwight",
+            last_name="D. Eisenhower",
+            email="dwight.d.eisenhower@uwblueprint.org",
+            auth_id="anothersecret",
+            role="User",
+            approved_languages_translation={"ENGLISH_US": 4, "ENGLISH_UK": 4},
+            approved_languages_review={"ENGLISH_US": 4, "ENGLISH_UK": 4},
+        ),
+    )
+
+
+def create_story_translation(db, story, translator, reviewer):
+    return db_session_add_commit_obj(
+        db,
+        StoryTranslation(
+            story_id=story.id,
+            language="ENGLISH_US",
+            stage="TRANSLATE",
+            translator_id=translator.id,
+            reviewer_id=reviewer.id,
+        ),
+    )
+
+
+def create_story_translation_contents(db, story_translation):
+    story_translation_content_1 = db_session_add_commit_obj(
+        db,
+        StoryTranslationContent(
+            story_translation_id=story_translation.id,
+            line_index=0,
+            translation_content="Translation content 1.",
+            status="APPROVED",
+        ),
+    )
+    story_translation_content_2 = db_session_add_commit_obj(
+        db,
+        StoryTranslationContent(
+            story_translation_id=story_translation.id,
+            line_index=1,
+            translation_content="",
+            status="DEFAULT",
+        ),
+    )
+    story_translation_contents = [
```

Comment:
hmmm this seems repetitive information and has space for human error, what about 
`story_translation_contents = [story_translation_content_1.__dict__, story_translation_content_2.__dict__]`

Reply:
Thanks for this nit (and the reviews)! I had to filter some keys because `story_translation_response["translation_contents"]` returns fewer keys per `translation_content` than `story_translation_content.to_dict()` does.